src/actions/resize/minimumPad.js

```diff
@@ -0,0 +1,25 @@
+import '../../interfaces/IAction';
+
+/**
+ * @implements IAction
+ */
+class MinimumPad {
+  /**
+   *
+   * @param {number} padWidth
+   * @param {number} padHeight
+   */
+  constructor(padWidth, padHeight) {
```

Comment:
Why not inherit from some general Resize class?

Reply:
Or implement a resize action interface?

Reply:
At this moment we don't have all the information of how everything looks like.
I think once we'll have more actions it'll be easier to consolidate.

There's no other Resize action here, so there's nothing to consolidate yet.

Reply:
Merging for now, we'll discuss reconsolidation once we have more code.